openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -306,6 +304,115 @@ def build_layout(project_name, asset_name):
             "jpg",
         )
 
+    # Setup cryptomatte
+    bpy.context.scene.view_layers[
+        "ViewLayer"
+    ].use_pass_cryptomatte_asset = True
+
+    # Load base nodegroup and make it publishable
+    _compo_container, compo_datablocks = load_subset(
+        project_name,
+        "CompositingNodesBank",
+        "nodegroupMatteColorCorrect",
+        "Append",
+    )
+
+    input_image_node = None
+    for container in containers:
+        if container.get("avalon", {}).get("family") == "rig":
+            input_image_node = setup_character_compositing(
+                asset_name,
+                container.name,
+                list(compo_datablocks)[0],
+                input_image_node,
+            )
+    else:
+        # Link last matte color correct node to composite node
+        composite_node = bpy.context.scene.node_tree.nodes["Composite"]
+        bpy.context.scene.node_tree.links.new(
+            input_image_node.outputs["Image"],
+            composite_node.inputs["Image"],
+        )
+
+
+def setup_character_compositing(
+    asset_name, character_name, source_compositing_nodegroup, input_image_node
+) -> bpy.types.NodeTree:
+    """Setup compositing nodes for character.
+
+    Args:
+        asset_name (str):  The current asset name from OpenPype Session.
+        character_name (str):  The character to create compositing of.
+        source_compositing_nodegroup (bpy.types.NodeTree):  The source nodegroup to use for compositing.
+        input_image_node (bpy.types.Node):  The input image node to use for compositing.
+
+    Returns:
+        bpy.types.NodeTree:  The output image node.
+    """
+    scene = bpy.context.scene
+    scene.use_nodes = True
+
+    # Get render layers node
+    render_layer_node = scene.node_tree.nodes.get("Render Layers")
+
+    # Set render layers node as input image node if not already set
+    if not input_image_node:
+        input_image_node = scene.node_tree.nodes["Render Layers"]
```

Comment:
So if `input_image_node` is `None`, it should have the same value as `render_layer_node`? I think `input_image_node = render_layer_node` would be simpler and faster then. Unless I'm missing something?